crates/bevy_text/src/font_atlas.rs

```diff
@@ -33,6 +36,33 @@ impl FontAtlas {
         self.glyph_to_index.get(&character).cloned()
     }
 
+    pub fn get_glyph_index(&self, glyph_id: GlyphId) -> Option<u32> {
+        self.glyph_to_atlas_index.get(&glyph_id).cloned()
+    }
+
+    pub fn has_glyph(&self, glyph_id: GlyphId) -> bool {
+        self.glyph_to_atlas_index.contains_key(&glyph_id)
+    }
+
+    pub fn add_glyph(
+        &mut self,
+        textures: &mut Assets<Texture>,
+        texture_atlases: &mut Assets<TextureAtlas>,
+        glyph_id: GlyphId,
+        texture: &Texture,
+    ) -> bool {
+        let texture_atlas = texture_atlases.get_mut(&self.texture_atlas).unwrap();
+        if let Some(index) =
+            self.dynamic_texture_atlas_builder
+                .add_texture(texture_atlas, textures, texture)
+        {
+            self.glyph_to_atlas_index.insert(glyph_id, index);
+            true
+        } else {
+            false
+        }
+    }
+
     pub fn add_char(
```

Comment:
i think we can remove this function (and `get_char_index()` and `glyph_to_index`)